app/src/models/team.model.ts

```diff
@@ -13,4 +13,6 @@ const Team = new Schema({
   createdAt: { type: Date, required: true, default: Date.now }
 });
 
-module.exports = mongoose.model("Team", Team);
+export const TeamModel = mongoose.model("Team", TeamSchema);
```

Comment:
Does this need to be an export? It's being exported as default in line 18

Reply:
I live re-exporting it in an index file: app/src/models/index.ts

so I can do `import { TeamModel } from "models"`

It's just preference, it's really neat when you have loads of files in a folder.

Still exporting as default because it's used elsewhere in old JS files